### ISSUE TITLE
[Feature] Is there an easy way to set up a character's relationship?

### ISSUE THREAD

**Issue by Laxtevin** [olivierkes/manuskript#821]
There are two characters,'A' and'B'.
And when a relationship was formed between the two.

I would like to be able to establish a relationship between the two simply by adding B in the item of A.
Up to this point, it is still possible, but it would be convenient if it was automatically added in B when it was set in A.

For example.
1. A loves B, and B hates A.
2. When trying to set for A, add B to the relationship (using drag or character list).
3. A->B love // B->A hate
4. When B is clicked, B->A hate // A->B love

I don't know if the delivery was correct because I was using the translator.

In short, when I take notes about human relationships, I hope it ends at once instead of working twice.

**Comment by obw** [comment]
Not yet, see my #144 issue

**Comment by Mannshoch** [comment]
See my Issue https://github.com/olivierkes/manuskript/issues/804
May could play a part in this cases.
